addons/l10n_dk_nemhandel/models/account_move.py

```diff
@@ -0,0 +1,69 @@
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    nemhandel_message_uuid = fields.Char(string='Nemhandel message ID')
+    nemhandel_move_state = fields.Selection(
+        selection=[
+            ('ready', 'Ready to send'),
+            ('to_send', 'Queued'),
+            ('skipped', 'Skipped'),
```

Comment:
```suggestion
```
That skipped state never made sense imo, I removed it later on:
https://github.com/odoo/odoo/commit/66b8413c6cddac3296d57e872cc25ba8dc1091a0#diff-14a573c641d57595927abfb4e72a786f836b6213f21eb07583efa255209be9d3R138